src/main/resources/stock_trades_schema.avro

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace": "ksql",
+  "name": "StockTrade",
+  "doc": "Defines a hypothetical stock trade using some known simulated stock symbols.",
+  "type": "record",
+  "fields": [
+    {
+      "name": "side",
+      "doc": "A simulated trade side (buy or sell or short)",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "BUY", "SELL", "SHORT" ]
+        }
+      }
+    },
+    {
+      "name": "quantity",
+      "doc": "A simualted random quantity of the trade",
+      "type": {
+        "type": "int",
+        "arg.properties": {
+          "range": {
+            "min": 1,
+            "max": 5000
+          }
+        }
+      }
+    },
+    {
+      "name": "symbol",
+      "doc": "Simulated stock symbols",
+      "type": {
+        "type": "string",
+        "arg.properties": {
+          "options": [ "ZJZZT","ZVZZT","ZWZZT" ]
```

Comment:
1. At first blush it's hard to differentiate these symbols.  Please change these to more distinguishable names.
2. Can you add more symbols?  It'll make for a more interesting UX in demos

Reply:
These are intentional.   They are industry known TEST symbols.  As I mentioned above, I don't like to leave room for interpretation that this is anything but simualtion.

Reply:
@rspurgeon so they are!  Can we add more per this page https://iextrading.com/trading/alerts/2017/027/ , e.g. CBO, CBX, ZBZX, ZJZZT, ZTEST, ZVV, ZVZZT, ZWZZT, ZXZZT ?

Reply:
Cool, nice find.  I'm good with adding the symbols that start with `Z`, but the `CBO` and `CBX` symbols resolve to tradable things on real exchanges so I'll leave those off.  I'll lookup each of the `Z*` symbols and add ones that come up TEST

Reply:
👍 💯

Reply:
Added